taxonomies/ingredients.txt

```diff
@@ -8369,20 +8422,22 @@ fr:lait de vache partiellement écrémé et pasteurisé
 
 <en:cow's milk
 en:pasteurised cow's milk
+ca:llet de vaca pasteuritzada, llet pasteuritzada de vaca
+da:Mælk (pasteuriseret)
```

Comment:
Are parentheses okay here?

Reply:
They are not, I removed the entry, it's just "milk" instead of "cow milk".